.github/workflows/pullrequest.yml

```
@@ -233,6 +233,48 @@ jobs:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'dirty bypass') }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          # Need full repo for Nx to find the commits at NX_* below
+          fetch-depth: 0
+          token: ${{ secrets.DIRTY_FIX_BOT_TOKEN }}
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Derive appropriate SHAs
+        if: ${{ inputs.run_merge_queue == false }}
```

actionlint (1.7.4)

property "run_merge_queue" is not defined in object type {}

(expression)